File: product-processor-service/src/main/java/com/example/product_processor_service/model/mail/dto/MailMessage.java
package com.example.product_processor_service.model.mail.dto;

import java.io.Serializable;

public class MailMessage implements Serializable {
    private String receiver;
    private String messageText;

    public MailMessage(String receiver, String messageText) {
        this.receiver = receiver;
        this.messageText = messageText;
    }

    public MailMessage() {
    }

    public String getReceiver() {
        return receiver;
    }

    public void setReceiver(String receiver) {
        this.receiver = receiver;
    }

    public String getMessageText() {
        return messageText;
    }

    public void setMessageText(String messageText) {
        this.messageText = messageText;
    }
}

File: email-sender-service/src/main/kotlin/com/example/email_sender_service/model/mail/dto/MailMessage.kt
package com.example.email_sender_service.model.mail.dto

class MailMessage {
    private lateinit var receiver: String
    private lateinit var messageText: String get set

    constructor(receiver: String, messageText: String) {
        this.receiver = receiver
        this.messageText = messageText
    }

    constructor() {

    }

    fun getReceiver() : String {
        return receiver
    }

    fun setReceiver(receiver: String) {
        this.receiver = receiver
    }

    fun getMessageText() : String {
        return messageText
    }

    fun setMessageText(messageText: String) {
        this.messageText = messageText
    }

}